lua/nvim-tree.lua

```diff
@@ -338,6 +338,19 @@ local function setup_autocommands(opts)
     })
   end
 
+  if opts.renderer.size.enable then
+    create_nvim_tree_autocmd({ "WinResized", "VimResized" }, {
+      -- NOTE: for some reason WinResized doesn't work with pattern
```

Comment:
This happens for a few autocommands. You can remove the comment.

Reply:
Oh, ok, ty for info